documentation/src/main/asciidoc/elasticsearch-integration.asciidoc

```diff
@@ -73,7 +73,7 @@ docker run -p 9200:9200 -d -v "$PWD/plugin_dir":/usr/share/elasticsearch/plugins
 ==== Dependencies in your Java application
 
 In addition to the usual dependencies like Hibernate ORM and Hibernate Search,
-you will need the new `hibernate-search-backend-elasticsearch` jar.
+you will need the new `hibernate-search-elasticsearch` jar.
```

Comment:
+1 for the need of a name, I keep calling it the "Lucene embedded vs Elasticsearch choice".

But whatever the name, I think we'll not want to add too many pre and postfixes to the Maven artifact so `hibernate-search-elasticsearch` looks good.